pipelines/batch/src/main/java/org/openmrs/analytics/FhirEtl.java

```diff
@@ -172,8 +172,8 @@ static void runFhirJdbcFetch(
         new JdbcConnectionUtil(
             options.getJdbcDriverClass(),
             dbConfig.makeJdbsUrlFromConfig(),
-            dbConfig.getDbUser(),
-            dbConfig.getDbPassword(),
+            dbConfig.getDatabaseUser(),
```

Comment:
Can we create the `JdbcConnectionUtil` object prior to `runFhirJdbcFetch` and `runHapiJdbcFetch` to remove duplication?

Reply:
We can but that adds another argument to these functions which I don't like. I created a simple method to refactor this.